test/ServiceBus.Tests/EvictionStrategyTests/EHPurgeLogicTests.cs

```diff
@@ -233,7 +230,15 @@ private Task AddDataIntoCache(EventHubQueueCacheForTesting cache, int count)
                 count--;
                 //just to make compiler happy
                 byte[] ignore = { 12, 23 };
-                cache.Add(new EventData(ignore), DateTime.UtcNow);
+                var eventData = new EventData(ignore);
+                DateTime now = DateTime.UtcNow;
+                var offSet = Guid.NewGuid().ToString() + now.ToString();
+                eventData.SetOffset(offSet);
+                //set sequence number
+                eventData.SetSequenceNumber(count);
+                //set enqueue time
+                eventData.SetEnqueuedTimeUtc(now);
+                cache.Add(eventData, DateTime.UtcNow);
             }
             return Task.CompletedTask;
```

Comment:
Why is this asynchronous?  Doesn't seem to need to be.

Reply:
I think I wanted to mimic 'concurrently adding items into caches' situation in the tests. In tests, items would be added to a list of cache in parallel. And the caches are sharing the same block pool . So I wanted to test at least in a minimum level that block pool handles 'concurrently adding items to cache' situation well.

Reply:
Understood, but c# is a bit to smart sometimes.  Because their is no real async work being done here, the logic will be run sequentially, and no parallelism will be generated.  Suggest adding a Task.Delay(1) to the start of this call, which will allow all tasks to be spun up prior to execution of core logic.

Reply:
That's good to know ! I didn't know that.  I will added it.